src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketProtocolTests.cs

```diff
@@ -108,6 +101,38 @@ public void CreateFromStream_ValidBufferSizes_Succeed(int bufferSize)
             }
         }
 
+        [Fact]
+        public static async Task ManagedWebSocket_ReceiveUTF8SplitAcrossMultipleBuffers()
+        {
+            // 1 character - 2 bytes
+            var payload = Encoding.UTF8.GetBytes("\u00E6");
+            var frame = new byte[payload.Length + 2];
+            frame[0] = 0x81; // FIN = true, Opcode = Text
+            frame[1] = (byte)payload.Length;
+            Array.Copy(payload, 0, frame, 2, payload.Length);
+
+            using (var stream = new MemoryStream())
+            {
+                stream.Write(frame);
+                stream.Seek(0, SeekOrigin.Begin);
+                stream.Flush();
+                var websocket = WebSocketProtocol.CreateFromStream(stream, false, "null", Timeout.InfiniteTimeSpan);
```

Comment:
Nit: var => WebSocket

Reply:
It's fine, but I'm curious why `"null"` instead of `null`?

Reply:
I originally passing in `""` and it complained telling me to pass in `"null"` so I did

Reply:
Ah, well "null" is ambiguous there :D